tox.ini

```diff
@@ -4,10 +4,25 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26, py27, py33, py34, pypy
+envlist = py26, py27, py33, py34, py36, pypy
```

Comment:
We might as well remove python 2.6 here as well. And from the trove classifiers in setup.py.

Reply:
Fixed